src/libraries/Microsoft.PowerFx.Core/Public/Values/OptionSetValue.cs

```diff
@@ -6,36 +6,65 @@
 using System.Diagnostics;
 using System.Text;
 using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Public.Types;
+using Microsoft.PowerFx.Core.Utils;
 
 namespace Microsoft.PowerFx.Core.Public.Values
 {
-    [DebuggerDisplay("OptionSetValue ({Option})")]
+    /// <summary>
+    /// A value within an option set. 
+    /// </summary>
+    [DebuggerDisplay("{ToString})")]
     public class OptionSetValue : FormulaValue
     {
         /// <summary>
         /// Logical name for this option set value.
         /// </summary>
-        public readonly string Option;
+        public string Option { get; private set; }
```

Comment:
Why not readonly anymore?

Reply:
field --> property. 
Still readonly semantics.

Reply:
Right, sorry. I meant, why the private set? I don't see that used anywhere.

Reply:
it's for the ctor on line 28.